.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
-version: 2
+version: 2.1
 gpu: &gpu
   machine:
     image: ubuntu-1604:201903-01
   resource_class: gpu.small
   environment:
     FPS_THRESHOLD: 900
-
+    CODECOV_TOKEN: c211f0c9-bc0c-46dc-8e2d-96d64a00a712
```

Comment:
Is it OK for this to be in source control>

Reply:
@msbaines, thanks for catching this. It should be CircleCI environment variable. I regenerated the token.